scripts/build/termux_step_get_dependencies.sh

```diff
@@ -1,7 +1,5 @@
 termux_step_get_dependencies() {
-	if [ "$TERMUX_SKIP_DEPCHECK" = true ] || [ "$TERMUX_PKG_METAPACKAGE" = true ]; then
-		return 0
-	fi
+	[ "$TERMUX_SKIP_DEPCHECK" = true ] && return
```

Comment:
Since this PR changed the method of processing dependencies during slow compilation of packages (i.e. when dependencies are compiled and not installed), it became relevant to process dependencies of meta-packages as well, since they (meta-packages) can be included in package dependencies.

FYI: @robertkirkman [reported](https://github.com/termux/termux-packages/pull/20513#issuecomment-2453586579) a problem compiling the `libepoxy` package using the builder from this PR. The problem turned out to be that the builder was skipping the `opengl` dependency check (the only dependency of `libepoxy`) since it is a meta package.